sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_service_client.hpp

```diff
@@ -104,16 +104,16 @@ namespace Azure { namespace Storage { namespace Blobs {
      * @brief Retrieves a key that can be used to delegate Active Directory authorization to
      * shared access signatures.
      *
-     * @param startsOn Start time for the key's validity, in ISO date format. The time should be
-     * specified in UTC.
-     * @param expiresOn Expiration of the key's validity, in ISO date format. The time should be
-     * specified in UTC.
+     * @param startsOn Start time for the key's validity. The time should be specified in UTC, and
+     * will be truncated to second.
```

Comment:
What motivated this doc change from ISO to UTC and what does it mean that it will be "truncated to second"?

Reply:
It's not changed. ISO and UTC are different kind of things.

In the past, we use `std::string` to represent the datetime, so the customer had to know the format (ISO or RFC).

Reply:
> what does it mean that it will be "truncated to second"?

fractional part will be truncated, like `1:23:45.6789000` will be truncated to `1:23:45`.